packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -273,4 +273,22 @@ describe('Dom Content', () => {
     cy.get('.passed > .num').should('contain', 1)
     cy.get('.failed > .num').should('contain', '--')
   })
+
+  describe('accessibility', () => {
+    it('has no axe violations in specs list panel', () => {
+      cy.visitApp()
+      cy.contains('withFailure.spec').click()
+      cy.get('button[aria-controls="reporter-inline-specs-list"]').click()
+      cy.injectAxe()
+      cy.checkA11y('[data-cy="specs-list-panel"]')
+    })
+
+    it('has no axe violations in reporter panel', () => {
+      cy.visitApp()
+      cy.contains('withFailure.spec').click()
+      cy.get('button[aria-controls="reporter-inline-specs-list"]').click()
+      cy.injectAxe()
+      cy.checkA11y('[data-cy="reporter-panel"]')
+    })
```

Comment:
Just to give you some context -- 

Ideally we would have a single test called `has no axe violations` that runs `cy.checkA11y()` with no targeted element. 

However, we've only recently started incorporating `cypress-axe` so if we were to do that, we would have lots of failures all at once 😸 In order to push the needle on a11y coverage without causing all our tests to fail, we scope `cy.checkA11y` accordingly. One day we'll be able to remove the targeted elements and just have that single test run.

Just wanted to give you a heads up so you know this isn't our ideal pattern long-term, and if you wanted you could combine these two axe tests into one